### PR TITLE
SB-24821 - Content-service local setup documentation update

### PR DESCRIPTION
**Pull request by shiva-rakshith**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-24821" title="SB-24821" target="_blank">Sub-taskSB-24821</a>  Content Service - Local setup documentation
  </td></table>
  <br />
 

SB-24821 - Content-service local setup documentation update

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Java 11, scala-2.11, play-2.7.2
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +96,8 @@ docker exec -it sunbird_cassandra cqlsh
 ```shell
 docker exec -it sunbird_cassandra /bin/bash
 ```
+5. Load seed data to cassandra using the instructions provided in the [link](https://github.com/project-sunbird/knowledge-platform/tree/master-data/master-data/loading-seed-data.md)
```
Comment (maheshkumargangula) [inline README.md:99]: Use relative path of the file.

File: README.md [hunk 2]
```diff
@@ -49,9 +49,11 @@ Using Docker on Windows will also need a couple of additional configurations bec
 
 By default, Neo4j requires authentication and requires us to first login with neo4j/neo4j and set a new password. We will skip this password reset by initializing the authentication none when we create the Docker container using the --env NEO4J_AUTH=none.
 
-3. After running the above command, neo4j instance will be created and container starts running, we can verify the same by accessing neo4j browser(http://localhost:7474/browser).
+3. Load seed data to neo4j using the instructions provided in the [link](https://github.com/project-sunbird/knowledge-platform/tree/master-data/master-data/loading-seed-data.md)
```
Comment (maheshkumargangula) [inline README.md:52]: Use relative path of the file every where.

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shiva-rakshith) [reply README.md]: Made suggested changes.

File: README.md (hunk 2 above)
Comment (shiva-rakshith) [reply README.md]: Made suggested changes.

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**